src/models/Item.js

```diff
@@ -4,11 +4,22 @@ export default class Item {
     this.ref = null;
   }
 
+  static getQuesionInfo(question) {
+    const imageMatch = question.match(/[\r\n]*\!\[.*\]\(.*=.*\)[\r\n]*/i);
+
+    const questionImage = imageMatch && imageMatch[0] || '';
+    const questionText = imageMatch ? question.substr(0, imageMatch.index) + question.substr(imageMatch.index + questionImage.length) : question;
```

Comment:
This can be simplified by storing the regular expression in a variable, so that you can do something like this:

```js
const imageRE = new RegExp(/[\r\n]*\!\[.*\]\(.*=.*\)[\r\n]*/i);
const imageMatch = question.match(imageRE);

const questionImage = imageMatch?[0] || '';  // The image URL.
const questionText = question.replace(imageRE, '');  // Remove the image from the question.
```